include/vapor/VDC.h

```diff
@@ -576,6 +576,16 @@ class VDF_API VDC : public VAPoR::DC {
     //! \sa DefineDimension(), DefineCoordVar(), SetCompressionBlock()
     //!
     int DefineDataVar(string varname, std::vector<string> dimnames, std::vector<string> coordvars, string units, XType type, double missing_value, string maskvar);
+    //!
+    //! Define a non-compressed data variable with missing data
+    //!
+    //!
+    //! \param[in] missing_value Specifies a value that should be used
+    //! for masked grid locations after a variable is reconstructed.
+    //!
+    //! \sa DefineDimension(), DefineCoordVar(), SetCompressionBlock()
+    //!
+    int DefineDataVar(string varname, std::vector<string> dimnames, std::vector<string> coordvars, string units, XType type, double missing_value);
```

Comment:
I think this overload is a little vague with the last parameter being completely different and it depends on the user using descriptive variable names. It also can lead to the ambiguous case of `DefineDataVar(..., 0)`. Does that mean a data var with compression disabled or a data var with a 0 missing value? This is a minor comment so feel free to reject if you disagree.

Reply:
Agreed. I've removed the newly added 3rd version of `DefineDataVar()` and replaced it with some minor logic changes to the original versions.